data/recipes/gcp_forensics.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "gcp_forensics",
   "short_description": "Copies disk from a GCP project to an analysis VM.",
-  "description": "Copies a disk from a project to another, creates an analysis VM, and attaches the copied disk to it.",
+  "description": "Copies a disk from a project to another, it will creates an analysis VM, and attaches the copied disk to it.",
```

Comment:
Why change this? All other sentences are in the present tense.

Reply:
changed it back